src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$clock-size: 660px;
+$border-radius: 50%;
+$color-yellow: #ff0;
```

Comment:
It's recommended to use more descriptive names for color variables, such as $color-primary, $color-secondary, or $color-accent, depending on their usage in the project.